server/s3_post_complete_action.cc

```diff
@@ -998,9 +997,15 @@ void S3PostCompleteAction::mark_old_oid_for_deletion() {
 }
 
 void S3PostCompleteAction::delete_old_object() {
+
+  if (S3Option::get_instance().is_s3server_obj_delayed_del_enabled()) {
```

Comment:
why this check
below we have check S3Option::get_instance()->is_s3server_obj_delayed_del_enabled())

Reply:
yes, reverting :)